### PR TITLE
podman 2.1.0

### PR DESCRIPTION
**Pull request by crunchtime-ali**
Created with `brew bump-formula-pr`.

---

[`action-homebrew-bump-formula`](https://github.com/dawidd6/action-homebrew-bump-formula)

### CODE REVIEW

**Review by ashley-cui — COMMENTED** [review]
Please change line 31 to 
```
assert_match "Error: Get", shell_output("#{bin}/podman 2>&1", 125)
```

This will make the check version-agnostic

File: Formula/podman.rb [hunk 1]
```diff
@@ -28,6 +28,6 @@ def install
 
   test do
     assert_match "podman version #{version}", shell_output("#{bin}/podman -v")
-    assert_match "Error: Get \"http://d/v1.0.0/libpod../../../_ping\"", shell_output("#{bin}/podman 2>&1", 125)
+    assert_match "Error: Get", shell_output("#{bin}/podman 2>&1", 125)
```
Comment (ashley-cui) [inline Formula/podman.rb:31]:
```suggestion
    assert_match "Error: Get", shell_output("#{bin}/podman info 2>&1", 125)
```

Comment (crunchtime-ali) [reply Formula/podman.rb]: Thank you for your suggestions @ashley-cui. I applied them without looking in to it because I did not have much time.

**Comment by crunchtime-ali** [comment]
`fatal: unable to access 'https://github.com/Homebrew/homebrew-core/': transfer closed with outstanding read data remaining
Error: Process completed with exit code 128.`

This looks to me like a network hickup. Can someone kick the build server, por favor?

File: Formula/podman.rb (hunk 1 above)
Comment (ashley-cui) [reply Formula/podman.rb]: No problem! Glad to help

**Comment by chenrui333** [comment]
> `fatal: unable to access 'https://github.com/Homebrew/homebrew-core/': transfer closed with outstanding read data remaining Error: Process completed with exit code 128.`
> 
> This looks to me like a network hickup. Can someone kick the build server, por favor?

it is github side issue, there is nothing that we can do.

**Comment by chenrui333** [comment]
And it should be all good now.

**Comment by chenrui333** [comment]
Thanks @crunchtime-ali and @ashley-cui!

**Pull request closed without merging**